wood-48x48/software/vcarve-post-processors/FCCH_LinuxCNC_Arcs_inch.pp

```diff
@@ -1,156 +1,156 @@
-+ =======================================

```

Comment:
Thanks for updating the docs!

Let's not rename these files; the filenames match what is actually on FCCH's PCs, and I believe was chosen to match the format of supplied pre-processors. Unix handles filenames that contain spaces fine; you just need quotes around them, or to escape the spaces, if using the command-line.

BTW, it's best to put changes like renames into a separate commit to separate "non-functional" changes from functional changes; it makes for smaller commits and makes review easier. In this case, I guess there are no changes to the file content during the rename?

Reply:
Good call -- I reverted the filenames back to their original form.